android/browserstack_sample.py

```diff
@@ -19,9 +16,9 @@
 
     # Set other BrowserStack capabilities
     'bstack:options' : {
```

Comment:
can remove 'bstack:options' let's only have generic caps here which might already be in user script before moving to bstack.

Reply:
removed